src/module_wrap.cc

```diff
@@ -449,7 +449,11 @@ MaybeLocal<Module> ModuleWrap::ResolveCallback(Local<Context> context,
                                                Local<String> specifier,
                                                Local<Module> referrer) {
   Environment* env = Environment::GetCurrent(context);
-  CHECK_NOT_NULL(env);  // TODO(addaleax): Handle nullptr here.
+  if (env == nullptr) {
+    Isolate* isolate = context->GetIsolate();
+    THROW_ERR_EXECUTION_ENVIRONMENT_NOT_AVAILABLE(isolate);
```

Comment:
There should probably be a `return;` after this line?

Reply:
@addaleax - done, returned a value following other error path scenarios.
[ also rebased to resolve a conflict ]